src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig

```diff
@@ -783,14 +783,14 @@
         class: attr.class ~ ' disabled'
       }) %}
     {% endif %}
-
-    {{ form_widget(form, {'attr': attr}) }}
+    {{- block('form_widget_simple') -}}
 
```

Comment:
Reason for this change is that label for file input(just after this comment) won't change it's contents to file name unless its directly below the input. And form_widget containts {{- block('form_help') -}} which inserted help block between input and it's label. As far as I checked `{{ form_widget(form, {'attr': attr}) }}` should be the same as `{{- block('form_widget_simple') -}}` for this case (aside for form_widget_simple not having help block inside).